src/nsrt_learning/option_learning.py

```diff
@@ -315,10 +316,11 @@ def _precondition_based_initiable(self, state: State, memory: Dict,
             # The memory here is used to store the absolute params, based on
             # the relative params and the object states.
             memory["params"] = params  # store for sanity checking in policy
```

Comment:
is this sanity check just verifying that the params here are always the same as when passed into the policy()? this is always guaranteed for any option in ANY situation, right?

Reply:
yeah